manifests/cinder-csi-plugin/cinder-csi-controllerplugin.yaml

```diff
@@ -54,6 +54,7 @@ spec:
             - "--timeout=3m"
             - "--default-fstype=ext4"
             - "--feature-gates=Topology=true"
+            - "--extra-create-metadata"
```

Comment:
Shouldn't we also bump the tag of image k8s.gcr.io/sig-storage/csi-provisioner?

Reply:
Its not required for this PR.  The flag `--extra-create-metadata` is present since 1.6.0, currently the manifests contain 2.1.1